test/users.js

```diff
@@ -449,3 +462,106 @@ describe('/users GET', () => {
       });
   });
 });
+
+describe('/admin/users POST', () => {
+  it('should pass creating administrator user, administrator token is provided and parameters are valid', () => {
+    // The users is created manually using the model to not depend on the
+    // Sign Up endpoint implementation
+    return User.create(someAdministratorUser)
```

Comment:
you could create a function that creates a user and call sign in so that you dont have to repeat this part over and over again. Also it could receive the user type as a parameter to use it in other tests too

Reply:
Can I do this on the next PR? Because there I have created a new file to use all the constants I use in tests. It would be a mess to do it here and then merge all in the other branch.